ProjectTemplates/templates/Worker-CSharp/Startup.cs

```diff
@@ -28,15 +31,31 @@ public void ConfigureServices(IServiceCollection services)
 
 #if (OrganizationalAuth)
             services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
+#if (GenerateApiOrGraph)
```

Comment:
Could this be structured without nesting all these #if clauses? smth like..

#if A
// code
#endif 
#if B
// code
#end B

Reply:
we try to remain as close as possible to the asp .net core templates. they are very picky w/their templates.